README.md

```diff
@@ -56,10 +56,45 @@
 -->
 [6]: https://codecov.io/gh/seqan/sharg-parser
 
-you can add this library to your application or include the [seqan3 library](https://github.com/seqan/seqan3)
-with even more bio c++ features.
+[7]: https://img.shields.io/badge/license-BSD-green.svg "Open Copyright page"
+<!--
+    This is the Copyright badge link:
+        `"Open Copyright page"` - this text will be shown on hover
+-->
+[8]: https://sharg.vercel.app/usr/html/about_copyright.html
+
+[9]: https://img.shields.io/badge/platform-linux%20%7C%20bsd%20%7C%20osx-informational.svg "Read more about our API"
```

Comment:
Why are all these links added and not used? [7] - [12]
It seems to me like it was intended to add the badges to the top (line 2).

Reply:
> In general, why are we naming the links just as numbers? I saw this in the seqan3 repository as well. You can name the links stuff like: [vercel_badge], [vercel_website], etc. It is more readable in my opition and it would prevent some errors, like the one with the [7] and [8] here.
> 
> Otherwise, nice README! :)

I have no idea, why we only use links as numbers, your proposal makes more sense. :)